spec/client_spec.rb

```diff
@@ -19,7 +19,10 @@
     it 'create a client with a missing api key' do
       expect {
         described_class.new(api_key: nil)
-      }.to raise_error(EasyPost::Error, 'API key is required.')
+      }.to raise_error(
+        EasyPost::Exceptions::MissingParameterError,
+        EasyPost::Constants::MISSING_REQUIRED_PARAMETER % 'api_key',
+      )
```

Comment:
Question: Is it possible to update lines 34 and 43 error types in this file since we define `EasyPost::TimeoutError` now?